auth0/resource_auth0_attack_protection.go

```diff
@@ -36,7 +37,9 @@ func newAttackProtection() *schema.Resource {
 							Description: "Whether or not breached password detection is active.",
 						},
 						"shields": {
-							Type: schema.TypeList,
+							Type:     schema.TypeSet,
+							Optional: true,
+							Computed: true,
```

Comment:
We're adding computed here and other places because this is an optional parameter that can have something set already from the API (sort of like a default). When this happens we need to have computed to true. From the docs:

"""
// Optional indicates whether the practitioner can choose to not enter
	// a value in the configuration for this attribute. Optional cannot be used
	// with Required.
	//
	// If also using Default or DefaultFunc, Computed should also be enabled,
	// otherwise Terraform can output warning logs or "inconsistent result
	// after apply" errors.
	Optional bool
"""